### DISCUSSION TITLE
WALC v0.3.3

### DISCUSSION THREAD

**Discussion by daffamumtaz2361** [cstayyab/WALC#241] [General]
## Bug fixes:
- Fix whatsapp integration

<hr /><em>This discussion was created from the release <a href='https://github.com/WAClient/WALC/releases/tag/'>WALC v0.3.3</a>.</em>

**Comment by albakhtari** [comment]
I confirm the non-ending loop of updating WhatsApp, I logged out (by deleting the cache) and then logged in again, but nothing changed.

I'm using KDE Plasma on Manjaro.

**Reply by albakhtari** [reply]
### It loops between these two screens:
![image](https://github.com/WAClient/WALC/assets/79008923/fa3f41c2-a823-4093-ab11-df5bf3a7275d)
![image](https://github.com/WAClient/WALC/assets/79008923/835605e5-7619-42ec-b3c2-d53eb6433e47)

**Reply by albakhtari** [reply]
Okay now it's working for some reason....

**Reply by albakhtari** [reply]
Okay, this is very weird, when I start walc using the command line it works flawlessly. But when I start it using Krunner or my menu it plays up.
Yes, I made sure that the .desktop file is executing the correct AppImage

**Comment by LostChanceee** [comment]
Thank you, that's project its savior my life.
